stock_kardex_report/wizard/stock_kardex_report_wizard.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2020, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, fields, models
+import textwrap
+
+
+class StockKardexReportWiz(models.TransientModel):
+    _name = 'stock.kardex.report.wiz'
+    _description = 'Wizard to create kardex reports of stock moves'
+
+    date_from = fields.Datetime(
+        string='From', required=True, default=fields.Datetime.now)
+    date_to = fields.Datetime(
+        string='To', required=True, default=fields.Datetime.now)
+    product = fields.Many2one('product.product', required=True)
+    location = fields.Many2one('stock.location', required=True)
+
+    def open_table(self):
+        self.env['stock.kardex.report'].search([]).unlink()
+        self._cr.execute('''
```

Comment:
You could have used double quotes here, to then cleanly use single quotes in the SQL.